test/FlapperUniV2.t.sol

```diff
@@ -85,55 +89,62 @@ contract FlapperUniV2Test is DssTest {
     address     SPOT;
-    address     DAI;
-    address     MKR;
+    address     USDS;
+    address     SKY;
     address     USDC;
-    address     LINK;
     address     PAUSE_PROXY;
     VatLike     vat;
     VowLike     vow;
+    address     UNIV2_USDS_IMX_PAIR;
 
     address constant LOG                 = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
 
+    address constant IMX                 = 0xF57e7e7C23978C3cAEC3C3548E3D615c346e79fF; // Random token that orders after USDS
     address constant UNIV2_FACTORY       = 0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f;
-    address constant UNIV2_DAI_MKR_PAIR  = 0x517F9dD285e75b599234F7221227339478d0FcC8;
-    address constant UNIV2_LINK_DAI_PAIR = 0x6D4fd456eDecA58Cf53A8b586cd50754547DBDB2;
+    address constant UNIV2_SKY_USDS_PAIR = 0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c;
 
     event Exec(uint256 lot, uint256 sell, uint256 buy, uint256 liquidity);
 
     function setUp() public {
         vm.createSelectFork(vm.envString("ETH_RPC_URL"));
 
-        DAI_JOIN      = ChainlogLike(LOG).getAddress("MCD_JOIN_DAI");
+        USDS_JOIN     = ChainlogLike(LOG).getAddress("USDS_JOIN");
         SPOT          = ChainlogLike(LOG).getAddress("MCD_SPOT");
-        DAI           = ChainlogLike(LOG).getAddress("MCD_DAI");
-        MKR           = ChainlogLike(LOG).getAddress("MCD_GOV");
+        USDS          = ChainlogLike(LOG).getAddress("USDS");
+        SKY           = ChainlogLike(LOG).getAddress("SKY");
         USDC          = ChainlogLike(LOG).getAddress("USDC");
-        LINK          = ChainlogLike(LOG).getAddress("LINK");
         PAUSE_PROXY   = ChainlogLike(LOG).getAddress("MCD_PAUSE_PROXY");
         vat           = VatLike(ChainlogLike(LOG).getAddress("MCD_VAT"));
         vow           = VowLike(ChainlogLike(LOG).getAddress("MCD_VOW"));
+
+        UNIV2_USDS_IMX_PAIR = Univ2FactoryLike(UNIV2_FACTORY).createPair(USDS, IMX);
         
-        splitter = new SplitterMock(DAI_JOIN);
+        splitter = new SplitterMock(USDS_JOIN);
         vm.startPrank(PAUSE_PROXY);
         vow.file("hump", 50_000_000 * RAD);
         vow.file("bump", 5707 * RAD);
         vow.file("flapper", address(splitter));
         vm.stopPrank();
 
-        (flapper, medianizer) = setUpFlapper(MKR, UNIV2_DAI_MKR_PAIR, 727 * WAD, "MCD_FLAP") ;
-        assertEq(flapper.usdsFirst(), true);
+        {
+            deal(IMX, UNIV2_USDS_IMX_PAIR, 200_000_0000 * WAD, true);
+            deal(USDS, UNIV2_USDS_IMX_PAIR, 10_000_0000 * WAD, true);
+            PairLike(UNIV2_USDS_IMX_PAIR).sync();
+        }
+
+        (flapper, medianizer) = setUpFlapper(SKY, UNIV2_SKY_USDS_PAIR, 727 * WAD, "MCD_FLAP") ;
+        assertEq(flapper.usdsFirst(), false);
 
-        (linkFlapper, linkMedianizer) = setUpFlapper(LINK, UNIV2_LINK_DAI_PAIR, 654 * WAD / 100, bytes32(0));
-        assertEq(linkFlapper.usdsFirst(), false);
+        (imxFlapper, imxMedianizer) = setUpFlapper(IMX, UNIV2_USDS_IMX_PAIR, 654 * WAD / 100, bytes32(0));
```

Comment:
Same comment as above: we could use a more realistic price for SKY (and IMX) to avoid confusion. The same applies to the SwapOnly test file.